src/objectTranslations/Hierarchy_Settings__c-fr.objectTranslation

```diff
@@ -1,194 +1,204 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomObjectTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
-  <fields>
```

Comment:
@skosgi1 I know you've had a fair bit of conflict resolution fun today, but I just wanted to callout that it looks like the changes in this file probably should not have happened.

Reply:
It looks like some placeholders are missing though after the translation merge - I wonder if those can be resolved in a separate PR. I'm going to spin up a quick org and pull down the metadata to see if I can resolve some of this.

Reply:
Thanks for catching this @robertbwatson I excluded the file from this PR.

Reply:
FYI I started PR #933 to fix translations - it seems like when you do a retrieve from the server, the metadata is quite a bit different.

Reply:
@robertbwatson what's the plan with #933 then. merge this and then #933 after. Merge #933 into this one? Mind clarifying?

Reply:
@lifewithryan @skosgi1 The plan is for #933 to be merged into master once our merge freeze is listed. I don't think there's anything left to do in this Terms Grades PR in regards to translations.